### PR TITLE
move pe number ingestion for setup into a migration

### PR DESCRIPTION
**Pull request by dandds**
This should take care of [this](https://www.pivotaltracker.com/story/show/160063695) bug. It moves PE number ingestion into a migration so that it will get run during our CD. I think both schema and state are fair game for migrations, and having the list of PE numbers is required initial state for the app. I got rid of related code in `PENumbers` that's not used anywhere else.

### CODE REVIEW

File: script/ingest_pe_numbers.py [hunk 1]
```diff
@@ -1,33 +0,0 @@
-from urllib.request import urlopen
```
[the remark this answers is not in the document]
Comment (dandds) [reply script/ingest_pe_numbers.py]: Sure, I put it back.

File: script/update [hunk 2]
```diff
@@ -7,8 +7,5 @@ source "$(dirname "${0}")"/../script/include/global_header.inc.sh
 # Enable DB migration
 MIGRATE_DB="true"
 
-# Run the shared update script
-source ./script/include/run_update
```
Comment (patricksmithdds) [inline script/update]: @dandds Was this removal intentional?

Comment (dandds) [reply script/update]: No... not sure how I managed that, will fix.

**Comment by dandds** [comment]
Need to rebase and resolve the migrations.
